### DISCUSSION TITLE
建议开发者和译者认真对待文档更新和翻译

### DISCUSSION THREAD

**Discussion by xinjiajuan** [AlistGo/alist#1319] [Q&A]
# 问题
- 当前版本`v2.6.1-0-geb15bce`新建S3账号界面与文档的内容完全脱节；
- 翻译不知所云，需要填写的参数和文档中介绍的参数差别巨大
- 文档的参数翻译敷衍严重，每个参数都没有合适的介绍，直接把英文名字翻译成中文？
# 例如
新建界面中中文名为`虚拟路径`，开始不知道是啥东西，文档也没写，啥叫虚拟路径啊？结果才搞明白，就是个账号名称而已吧，什么叫路径？非常奇怪。
# 建议
希望开发者或者译者能认真对待界面文字和文档介绍的统一！

**Comment by xinjiajuan** [comment]
![image](https://user-images.githubusercontent.com/36360150/178096308-4ecd5a2a-1445-45f1-aab3-d3b8076b8a62.png)
另外docker安装方法要是按照你文档的方法，不就直接不晓得密码登不进去了吗，结果你介绍初始密码在hub.docker.com才能看到，你文档是被你吃了吗？

**Reply by joshgad** [reply]
u Don't have to use it if u don't want to. discussion is a  place of peaceful communication, OK?

**Reply by kxs2018** [reply]
> u Don't have to use it if u don't want to. discussion is a place of peaceful communication, OK?

牛逼啊，别人给你提建议直接叫人不要用了

**Reply by aobozhang** [reply]
> > u Don't have to use it if u don't want to. discussion is a place of peaceful communication, OK?
> 
> 牛逼啊，别人给你提建议直接叫人不要用了

还请不要到别人家里排……泄，并且舔着恩赐脸。
如果秉持善意，愿您可以共同维护文档并提交PR。

**Comment by xiaowang7212** [comment]
开发者不更新喽·

**Comment by hbqjren** [comment]
开发者很忙，也很累。请你先去看下中文说明书

**Comment by Limourli-liu** [comment]
emm，语言可以更委婉一些，别人也是辛苦开源的

**Comment by Limourli-liu** [comment]
另外，文档里是有视频的，至少OneDrive的挂载作者的视频介绍的相当清楚，我这样的小白都能按视频完成挂载。
